File: 1_Two_Sum/main.go
func twoSum(nums []int, target int) []int {
	arr := make(map[int]int)
	ans := []int{}

	for i, num := range nums {
		if _, ok := arr[num]; ok {
			ans = append(ans, i)
			ans = append(ans, arr[num])
			break
		}
		x := target - num
		arr[x] = i
	}

	return ans
}

File: 1_Two_Sum/main.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        arr = {}

        for i in range(len(nums)):
            if nums[i] in arr:
                return [i, arr[nums[i]]]
            arr[target - nums[i]] = i
